queries.sql

```diff
@@ -38,4 +38,28 @@ WHERE name != 'Gabumon'
 /*  Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) */
 
 SELECT * FROM animals
-WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 AND weight_kg = 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 AND 17.3 OR weight_kg = 10.4 AND weight_kg = 17.3;
+
+
+/* How many animals are there? */
+SELECT COUNT(*) FROM animals;
+
+/* How many animals have never tried to escape? */
+SELECT COUNT(*) FROM animals
+WHERE escape_attempts = 0;
+
+/* What is the average weight of animals? */
+SELECT AVG(weight_kg) FROM animals;
+
```

Comment:
* The queries for transactions stated in the project [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction,Commit%20transaction) are missing kindly add them 🙏